### ISSUE TITLE
new patch 9.2.5 is bugged

### ISSUE THREAD

**Issue by reyard** [romdeau23/wow-lfgspamfilter#3]
New pach I don't see the addon.

**Comment by Sipollo** [comment]
There's a LUA error in the addon's `init()` function which is causing it not to load:

```
LFGSpamFilter\modules/main.lua:7: hooksecurefunc(): ReportSearchResult is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "@LFGSpamFilter\modules/main.lua"]:7: in function `init'
[string "@LFGSpamFilter\LFGSpamFilter-1.0.0.lua"]:29: in function <LFGSpamFilter\LFGSpamFilter.lua:23>
```

**Comment by Skullcan** [comment]
Looks like the function C_LFGList.ReportSearchResult(resultID, complaintType) of the WoW API has been changed.

Blizzard updated the 'report tool' to include other types of reports, this might have changed something.

source:
https://us.forums.blizzard.com/en/wow/t/an-eye-on-player-behavior-and-reporting-improvements/1249578/1

**Comment by romdeau23** [comment]
Also `C_ReportSystem.SendReport()` is protected, so this addon can no longer report groups by itself. Will release an update soon.

**Comment by romdeau23** [comment]
Should be fixed in e9f29452dc1
